src/__internal__/focus-trap/focus-trap.component.js

```diff
@@ -13,19 +14,21 @@ import {
   isRadio,
   setElementFocus,
 } from "./focus-trap-utils";
+import { ModalContext } from "../../components/modal/modal.component";
 
 const FocusTrap = ({
   children,
   autoFocus = true,
   focusFirstElement,
   bespokeTrap,
   wrapperRef,
+  isAnimationComplete = false,
 }) => {
   const firstOpen = useRef(true);
   const [focusableElements, setFocusableElements] = useState();
   const [firstElement, setFirstElement] = useState();
   const [lastElement, setLastElement] = useState();
-
+  const modalContext = useContext(ModalContext);
```

Comment:
You could destructure the value from the context if you wanted

```suggestion
  const { isAnimationComplete } = useContext(ModalContext);
```